code/modules/clothing/head/misc.dm

```diff
@@ -80,11 +80,12 @@
 	body_parts_covered = FULL_HEAD|BEARD
 	item_state="cueball"
 
-/obj/item/clothing/head/that
```

Comment:
There's a second 'that' at line 24 -- this was always supposed to be an armored top hat, old coders just didn't implement it right.